Makefile

```diff
@@ -142,7 +142,7 @@ E2E_SKIP_EKS_UPGRADE ?= "false"
 EKS_SOURCE_TEMPLATE ?= eks/cluster-template-eks-control-plane-only.yaml
 
 # set up `setup-envtest` to install kubebuilder dependency
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.24.2
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.28.3
```

Comment:
Should this be 1.29.1?

Reply:
Nope, in CAP* we follow the k8s release that core CAPI is bases off of.
For CAPI 1.6.0 it is 1.28, and 1.28.3 is the latest envtest version up in the kubebuilder-tools GCS bucket for 1.28.

Reply:
Thanks, I noticed that everything else was 1.28 as I continued to read.